src/client/components/filter-options-dropdown/filter-options-dropdown.scss

```diff
@@ -45,6 +52,9 @@
   }
 
   svg.icon {
+    @include internet-explorer {
+      height: 29px;
```

Comment:
IE have problems with sag heights, but should all browsers accept those settings correctly? We could leave height for all browsers here.